README.md

```diff
@@ -106,7 +107,26 @@ In your CSS you need to add the following:
 ```
 
 And that's it! Your file input is now styled and decked to the nines!
-If you would like to see a real [life example checkout the demo page](http://thefrontside.github.io/emberx-file-input)
+If you would like to see a real
+[life example checkout the demo page](http://thefrontside.github.io/emberx-file-input)
+
+## Resetting the input
+
+If you would like to select the same file many times you need to call
```

Comment:
Two paragraphs starting with `If you would like` is one to many. Let's have the second say "To select the same file many times..."